### ISSUE TITLE
2 errors in 0.7.1 installed from pypi

### ISSUE THREAD

**Issue by emdee-is** [metebalci/digsec#3]
You need to run pylint on your code as a part of release QA:
```
************* Module digsec.algorithms
algorithms.py:59:11: E0602: Undefined variable 'BadSignatureError' (undefined-variable)
algorithms.py:72:11: E0602: Undefined variable 'BadSignatureError' (undefined-variable)
************* Module digsec.messages
messages.py:250:4: E0213: Method 'from_packet' should have "self" as first argument (no-self-argument))
```
from_packet needs a staticmethod and BadSignatureError needs adding  to the from ecdsa import 

Your code is great and is just what I need, but a little basic documentation would help people like me who have never used DNSSEC. Some more explanation in the ```validate_second_level_domain.sh``` script of the steps, and the importance of the order of the sequence of steps, would help a lot.

**Comment by metebalci** [comment]
I will add pylint

**Comment by metebalci** [comment]
fixed with e0aedd23cb73b695c9b364ac124f0c9556be59c4
